tools/test/deployment_tools.test.js

```diff
@@ -0,0 +1,164 @@
+const Contract = require('../deployment_tools.js').Contract;
+const ContractRegistry = require('../deployment_tools.js').ContractRegistry;
+const IncrementingNonceAddressGenerator = require('../deployment_tools.js').IncrementingNonceAddressGenerator;
+
+const EIP20TokenFile = require('../../build/contracts/EIP20Token.json');
+
+const rootDir = `${__dirname}/../../`;
+
+describe('Contract', () => {
+    describe('.setAddress()', () => {
+        test('correctly sets a fixed address for contract', () => {
```

Comment:
We usually use mocha `it` as per truffle. Is that an option here?

Reply:
We can definitely switch it to mocha + chai 👍 . I just picked Jest as I was most used to it and there wasn't a general JS testing framework in place.